Source/DafnyLanguageServer/Workspace/VerificationProgressReporter.cs

```diff
@@ -133,13 +137,14 @@ INotificationPublisher notificationPublisher
           if (subsetTypeDecl.tok.Filename != documentFilePath) {
             continue;
           }
-          var verificationTreeRange = subsetTypeDecl.tok.GetLspRange(subsetTypeDecl.BodyEndTok);
+          var verificationTreeRange = subsetTypeDecl.StartToken.GetLspRange(subsetTypeDecl.EndToken);
           var verificationTree = new TopLevelDeclMemberVerificationTree(
             $"subset type",
             subsetTypeDecl.Name,
             subsetTypeDecl.CompileName,
             subsetTypeDecl.tok.Filename,
-            verificationTreeRange);
+            verificationTreeRange,
+            subsetTypeDecl.tok.GetLspPosition());
```

Comment:
(When) will these positions be different from the first token of the `verificationTreeRange`?

Reply:
I'm going to explain it in an example in a new PR, but typically, the tok is the symbol name, whereas the range extends to the left (keywords) and to the right of this symbol name (definition)